source/includes/versioned-api-snippets/scala-client.scala

```diff
@@ -0,0 +1,13 @@
+import com.mongodb.{ServerApi, ServerApiVersion}
+import org.mongodb.scala._
+
+// ...
```

Comment:
The indentation is off here

Reply:
I will modify the rest of the code to be aligned with the first column of the line.

I can see how the indentation is confusing.
 
I've used "// ..." to denote "the rest of your code required to make this work" which means you'd probably want to have the class definition without indentation and then all the code within it in a function inside, so actually I'd want double indentation.
Instead, I think it's easier to align everything to the left so we're left with just the imports and section of the relevant code. Hopefully, we can get a collapsible code block element so that we can show both the full code and relevant code.